docker-compose.yaml

```diff
@@ -6,7 +6,7 @@ services:
     platform: linux/amd64
     build:
       context: .
-      dockerfile: ./build/Dockerfile
```

Comment:
if the users are going to leverage the readme for a container build process, and build/Dockerfile is the one that's pinned to a version, i think this should stay build/Dockerfile right? let's sync on all the pieces in the morning so i'm sure i know how the story works.

Reply:
As discussed, this will the user community version on `./build`. Thanks for the input.